test/integration/deploy_int_test.go

```diff
@@ -155,9 +155,11 @@ func (suite *DeployIntegrationTestSuite) TestDeployPython() {
 		suite.T().Skipf("Skipping DeployIntegrationTestSuite when not running on CI, as it modifies bashrc/registry")
 	}
 
-	ts := e2e.New(suite.T(), false)
+	ts := e2e.New(suite.T(), true)
```

Comment:
Is this intentional? Or was it a residual change from testing/debugging?

Reply:
Whoops, that's a leftover from debugging.